js-miniapp-sample/src/pages/universal-bridge.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import MiniApp from 'js-miniapp-sdk';
+import { HostAppEvents } from 'js-miniapp-sdk';
+
+import {
+  Button,
+  TextField,
+  CardContent,
+  CardActions,
+  makeStyles,
+} from '@material-ui/core';
+
+import GreyCard from '../components/GreyCard';
+
+const useStyles = makeStyles((theme) => ({
+  scrollable: {
+    overflowY: 'auto',
+    width: '100%',
+    paddingTop: 20,
+    paddingBottom: 20,
+  },
+  card: {
+    width: '100%',
+    height: 'auto',
+  },
+  actions: {
+    justifyContent: 'center',
+  },
+  content: {
+    justifyContent: 'center',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    fontSize: 18,
+    color: theme.color.primary,
+    fontWeight: 'bold',
+    paddingBottom: 0,
+  },
+  textfield: {
+    width: '90%',
+    maxWidth: 300,
+    marginTop: 10,
+    marginBottom: 10,
+    background: 'white',
+    '& input': {
+      color: theme.color.primary,
+      lineHeight: '1.5em',
+      fontSize: '1.2em',
+    },
+  },
+}));
+
+const UniversalBridge = () => {
+  const classes = useStyles();
+  const defaultJsonValue = '{"data":"This is a sample json information"}';
+  let inputValue = defaultJsonValue;
+  let [receiveJsonInfo, setReceiveJsonInfo] = useState('');
+
+  window.addEventListener(HostAppEvents.RECEIVE_JSON_INFO, function (e) {
+    let message = e.detail.message;
+    console.log(message);
+    receiveJsonInfo = message;
+    setReceiveJsonInfo(message);
+  });
+
+  const handleInput = (e: SyntheticInputEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    inputValue = e.currentTarget.value;
+  };
+
+  const sendJson = () => {
+    const info = { content: inputValue };
+    MiniApp.universalBridge
+      .sendJsonToHostapp(info)
+      .then((success) => {
+        console.log(success);
```

Comment:
We need add an alert/label which shows the success or error message

Reply:
Yeah, I have added the alerts for the success or error message.